client/src/components/Registration/Registration.tsx

```diff
@@ -6,14 +6,16 @@ const Registration = () => {
   const [registerEmail, setRegisterEmail] = useState("");

```

Comment:
It should be better to use "SignUp" instead of "Registration" as we have other registration forms and might be confusing which one is for what. We should also use the React function declaration syntax Ex: `const FunctionName: React.FC = () => {}`